build error for yaz 5.34.0 with libxml2 2.13.0

With libxml2 2.12.0, yaz 5.34.0 fails to build as:

```
record_render.c:231:9: error: implicit declaration of function 'xmlKeepBlanksDefault' [-Werror=implicit-function-declaration]
record_render.c:232:15: error: implicit declaration of function 'xmlParseMemory' [-Werror=implicit-function-declaration]
yaz-icu.c:437:19: error: implicit declaration of function 'xmlParseFile'; did you mean 'xmlSaveFile'? [-Werror=implicit-function-declaration]
yaz-record-conv.c:44:33: error: implicit declaration of function 'xmlParseFile'; did you mean 'xmlSaveFile'? [-Werror=implicit-function-declaration]
cclsh.c:70:19: error: implicit declaration of function 'xmlParseFile'; did you mean 'xmlSaveFile'? [-Werror=implicit-function-declaration]
test_ccl.c:201:19: error: implicit declaration of function 'xmlParseMemory' [-Werror=implicit-function-declaration]
test_icu.c:452:19: error: implicit declaration of function 'xmlParseMemory' [-Werror=implicit-function-declaration]
test_xml_include.c:27:11: error: implicit declaration of function 'xmlParseMemory' [-Werror=implicit-function-declaration]
client.c:902:9: error: implicit declaration of function 'xmlKeepBlanksDefault' [-Werror=implicit-function-declaration]
client.c:903:15: error: implicit declaration of function 'xmlParseMemory' [-Werror=implicit-function-declaration]
```

--- a/client/client.c
+++ b/client/client.c
@@ -73,6 +73,10 @@
 #include <readline/history.h>
 #endif
 
+#if YAZ_HAVE_XML2
+#include <libxml/parser.h>
+#endif
+
 
 #include "admin.h"
 #include "tabcomplete.h"

--- a/src/record_render.c
+++ b/src/record_render.c
@@ -23,6 +23,7 @@
 #include <yaz/base64.h>
 
 #if YAZ_HAVE_XML2
+#include <libxml/parser.h>
 #include <libxml/xpath.h>
 #include <libxml/xpathInternals.h>
 #endif

--- a/test/test_ccl.c
+++ b/test/test_ccl.c
@@ -11,6 +11,9 @@
 #include <yaz/log.h>
 #include <yaz/test.h>
 
+#if YAZ_HAVE_XML2
+#include <libxml/parser.h>
+#endif
 
 static int tst_ccl_query(CCL_bibset bibset,
                          const char *query,

--- a/test/test_icu.c
+++ b/test/test_icu.c
@@ -28,6 +28,7 @@
 #endif
 
 #if YAZ_HAVE_XML2
+#include <libxml/parser.h>
 #include <libxml/xmlmemory.h>
 #endif
 

--- a/test/test_xml_include.c
+++ b/test/test_xml_include.c
@@ -12,6 +12,9 @@
 
 #include <yaz/xml_include.h>
 #include <yaz/test.h>
+#if YAZ_HAVE_XML2
+#include <libxml/parser.h>
+#endif
 
 static void tst_xml_include1(void)
 {

--- a/util/cclsh.c
+++ b/util/cclsh.c
@@ -20,6 +20,9 @@
 #include <readline/history.h>
 #endif
 
+#if YAZ_HAVE_XML2
+#include <libxml/parser.h>
+#endif
 
 static int debug = 0;
 static char *prog;

--- a/util/yaz-icu.c
+++ b/util/yaz-icu.c
@@ -17,6 +17,8 @@
 
 #if YAZ_HAVE_ICU
 
+#include <libxml/parser.h>
+
 #include <unicode/ucnv.h>
 #include <unicode/ustring.h>
 #include <unicode/ucol.h>

--- a/util/yaz-record-conv.c
+++ b/util/yaz-record-conv.c
@@ -13,6 +13,10 @@
 #include <yaz/record_conv.h>
 #include <yaz/backtrace.h>
 
+#if YAZ_HAVE_XML2
+#include <libxml/parser.h>
+#endif
+
 const char *prog = "yaz-record-conv";
 
 static void usage(void)

